build.gradle

```diff
@@ -46,15 +46,16 @@ dependencies {
     compile "com.android.support:design:${supportLibraryVersion}"
     compile 'com.jakewharton:disklrucache:2.0.2'
     compile 'com.google.android.exoplayer:exoplayer:r2.2.0'
+    compile 'com.andrognito.patternlockview:patternlockview:1.0.0'
```

Comment:
When we use a third-party library we usually include the information in our [THIRD_PARTY.txt](https://github.com/owncloud/android/blob/master/THIRD_PARTY.txt) file. Could you do the same with this library? You can have a look at the format we followed with other libraries in that file.

Reply:
As in this picture for the floating action button dependency it is written that it is placed in libs/com-getbase-floatingactionbutton-1-10-0-exploded-aar. Similarly what should I write for the placed at point (for the pattern lock view dependency).What I mean is what does placed at signify.</br>
<img width="1013" alt="screen shot 2018-02-05 at 9 23 36 pm" src="https://user-images.githubusercontent.com/22047421/35813943-e3c50840-0aba-11e8-8d7a-2cd542fe6ab8.png">

Reply:
Forget this change for the moment, it seems that our THIRD_PARTY.txt is not properly updated, I will ping you when is ready.

Reply:
@davigonz Then I will commit the other changes. Thanks.